arches_references/templates/javascript.htm

```diff
@@ -14,22 +14,4 @@
     multiple-values='{% trans "Multiple Values" as multipleValues %} "{{ multipleValues|escapejs }}"'
     allow-selection-of-multiple-items='{% trans "Allow selection of multiple items" as allowSelectionOfMultipleItems %} "{{ allowSelectionOfMultipleItems|escapejs }}"'
 ></div>
-{% endblock arches_translations %}
-
-
-{% block arches_urls %}
-{{ block.super }}
-<div class="arches-urls"
-    controlled_lists="{% url 'controlled_lists' %}"
-    controlled_list='(listid) => {return "{% url "controlled_list" "aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa" %}".replace("aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa", listid)}'
-    controlled_list_add="{% url 'controlled_list_add' %}"
-    controlled_list_item='(itemid) => {return "{% url "controlled_list_item" "aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa" %}".replace("aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa", itemid)}'
-    controlled_list_item_add="{% url 'controlled_list_item_add' %}"
-    controlled_list_item_value='(valueid) => {return "{% url "controlled_list_item_value" "aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa" %}".replace("aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa", valueid)}'
-    controlled_list_item_value_add="{% url 'controlled_list_item_value_add' %}"
-    controlled_list_item_image='(imageid) => { return "{% url "controlled_list_item_image" "aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa" %}".replace("aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa", imageid)}'
-    controlled_list_item_image_add="{% url 'controlled_list_item_image_add' %}"
-    controlled_list_item_image_metadata='(metadataid) => { return "{% url "controlled_list_item_image_metadata" "aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa" %}".replace("aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa", metadataid)}'
-    controlled_list_item_image_metadata_add="{% url 'controlled_list_item_image_metadata_add' %}"
-></div>
-{% endblock arches_urls %}
\ No newline at end of file
+{% endblock arches_translations %}
```

Comment:
The 🚫 is here because this file lacks a newline at the end.

Our project template may not be very careful about this, but for new files of our own, let's be sure to use them. Just FYI for future--not worth changing them ATM.